default/stringtables/en.txt

```diff
@@ -1893,6 +1893,9 @@ OPTIONS_DB_UI_PLANET_STATUS_CONQUERED
 %1% has just been conquered. Stability will probably be reduced.
 
 OPTIONS_DB_UI_PLANET_STATUS_UNHAPPY
+The population on %1% is unhappy. Both [[metertype METER_INDUSTRY]] and [[metertype METER_RESEARCH]] will grow slowly. It can not be used as a source for settlers.
```

Comment:
Shouldn't it say "is unstable"? Same for the entry below.

Perhaps after the release we should replace all occurrences of "happy" and "happiness" also in variable/macro names to get rid of this mess once and for all.

Reply:
> Perhaps after the release we should replace all occurrences of "happy" and "happiness" also in variable/macro names to get rid of this mess once and for all.

I strongly support this suggestion.

Reply:
@MichaelCourtney poke

Reply:
I'll go through it and change them all. I suspect some usages of "is unstable" will read as "is mentally unstable". Though that's hard to avoid.